ingestion/src/metadata/ingestion/source/dashboard/redash.py

```diff
@@ -121,60 +128,75 @@ def yield_dashboard_lineage_details(
-                table_name = str(table)
-                database_schema = None
-                if "." in table:
-                    database_schema, table = fqn.split(table_name)[-2:]
-                table_entities = search_table_entities(
-                    metadata=self.metadata,
-                    database=None,
-                    service_name=db_service_name,
-                    database_schema=database_schema,
-                    table=table_name,
-                )
-                for from_entity in table_entities:
-                    to_entity = self.metadata.get_by_name(
-                        entity=Lineage_Dashboard,
-                        fqn=fqn.build(
-                            self.metadata,
-                            Lineage_Dashboard,
-                            service_name=self.config.serviceName,
-                            dashboard_name=str(dashboard_details.get("id")),
-                        ),
+
+        try:
+            for widgets in dashboard_details.get("widgets", []):
+                visualization = widgets.get("visualization")
+                if not visualization.get("query"):
+                    continue
+                if visualization.get("query", {}).get("query"):
+                    parser = LineageRunner(
+                        clean_raw_query(visualization["query"]["query"])
                     )
-                    yield self._get_add_lineage_request(
-                        to_entity=to_entity, from_entity=from_entity
+                for table in parser.source_tables:
+                    table_name = str(table)
+                    database_schema = None
+                    if "." in table:
+                        database_schema, table = fqn.split(table_name)[-2:]
+                    table_entities = search_table_entities(
+                        metadata=self.metadata,
+                        database=None,
+                        service_name=db_service_name,
+                        database_schema=database_schema,
+                        table=table_name,
                     )
+                    for from_entity in table_entities:
+                        to_entity = self.metadata.get_by_name(
+                            entity=Lineage_Dashboard,
+                            fqn=fqn.build(
+                                self.metadata,
+                                Lineage_Dashboard,
+                                service_name=self.config.serviceName,
+                                dashboard_name=str(dashboard_details.get("id")),
+                            ),
+                        )
+                        yield self._get_add_lineage_request(
+                            to_entity=to_entity, from_entity=from_entity
+                        )
+        except Exception as exc:
+            logger.debug(traceback.format_exc())
+            logger.warning(
+                f"Error to creating dashboard chart for {dashboard_details}: {exc}"
```

Comment:
Please change the message to `Error to yield dashboard lineage details for DB service name [{db_service_name}]: {err}`